lib/connect-session-sequelize.js

```diff
@@ -44,7 +45,9 @@ module.exports = function SequelizeSessionInit (Store) {
       } else {
         // No Table specified, default to ./model
         debug('No table specified, using default table.')
-        this.sessionModel = options.db.import(path.join(__dirname, 'model'))
+        let modelKey = options.defaultModelKey || "Session" // Camelcase is the Standard-key. This should 
```

Comment:
```suggestion
        const modelKey = options.defaultModelKey || "Session" // Camelcase is the Standard-key. This should 
```

Reply:
We can take the change if you want. I wanted the "variable" only for this block but maybe const has some benefits?